fre/cmor/cmor_mixer.py

```diff
@@ -166,12 +169,23 @@ def rewrite_netcdf_file_var ( proj_table_vars = None,
                               target_var = None,
                               json_exp_config = None,
                               json_table_config = None):#, tmp_dir = None            ):
-    ''' rewrite the input netcdf file nc_fl containing target_var in a CMIP-compliant manner.
+    ''' 
+    rewrite the input netcdf file nc_fl containing target_var in a CMIP-compliant manner.
+    accepts six arguments, all required:
+        proj_table_vars: json dictionary object, variable table read from json_table_config.
+        local_var: string, variable name used for finding files locally containing target_var,
+                   this argument is often equal to target_var.
+        netcdf_file: string, representing path to intput netcdf file.
+        target_var: string, representing the variable name attached to the data object in the netcdf file.
+        json_exp_config: string, representing path to json configuration file holding metadata for appending to output
+                         this argument is most used for making sure the right grid label is getting attached to the right output
+        json_table_config: string, representing path to json configuration file holding variable names for a given table.
+                           proj_table_vars is read from this file, but both are passed anyways. 
```

Comment:
OK, so local_var is more clearly related to the variable renaming feature (t_ref => tas), I see.

Reply:
yes! that's the idea as i've come to understand it, and initially needed to even be convinced of this by you! I tried to clarify this functionality with `fre/tests
/test_fre_cmor_cli.py::test_cli_fre_cmor_run_case2` and `fre/cmor/tests
/test_cmor_run_subtool.py::test_fre_cmor_run_subtool_case2`